doc/neptune-metadata.rst

```diff
@@ -0,0 +1,105 @@
+===========================
+Neptune Metadata Definition
+===========================
+
+This file specifies what fields are in the ``.neptune.yml`` file, what they mean
+and by which tools they are used. The file itself is expected to be in a
+repositories root directory.
+
+.. contents::
+
```

Comment:
Maybe add a line somewhere that explains what the following sections describe.

Reply:
But.. it says it right at the top.. "This file specifies what fields [..]"?

Reply:
Haha, oh yes, I seem to have missed that ;)